lftpd.c

```diff
@@ -285,6 +294,12 @@ static int cmd_cwd(lftpd_client_t *client, const char *arg)
 
 static int cmd_dele(lftpd_client_t *client, const char *arg)
 {
+	if (!client->authenticated)
+	{
+		send_simple_response(client->socket, 530, "Not logged in.");
+		return 0;
```

Comment:
Make the code so that functions have only one return statement.
Also should we return an error code?